Lib/asyncio/base_subprocess.py

```diff
@@ -104,7 +104,12 @@ def close(self):
         for proto in self._pipes.values():
             if proto is None:
                 continue
-            proto.pipe.close()
+            # See https://github.com/python/cpython/issues/114177
```

Comment:
I think a gh-NNNNN reference is enough.

```suggestion
            # See gh-114177
```